pkgs/development/python-modules/pydyf/default.nix

```diff
@@ -19,6 +20,15 @@ buildPythonPackage rec {
     sha256 = "sha256-Hi9d5IF09QXeAlp9HnzwG73ZQiyoq5RReCvwDuF4YCw=";
   };
 
+  patches = [
+    # Fix tests for Ghostscript 9.56
+    # Remove after v0.1.3 has been released
+    (fetchpatch {
+      url = "https://github.com/CourtBouillon/pydyf/commit/d4c34823f1d15368753c9c26f7acc7a24fc2d979.patch";
+      sha256 = "sha256-Fx5I/8nRG6ULWiTV12WLeiJp9XPLm2E2QiNLy86MJB4=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/kmkbv2mzzwnr6ryjh7mhrvb9f0ivqahv-d4c34823f1d15368753c9c26f7acc7a24fc2d979.patch.drv':
         specified: sha256-Fx5I/8nRG6ULWiTV12WLeiJp9XPLm2E2QiNLy86MJB4=
            got:    sha256-2hHZW/q5CbStbpSJYbm3b23qKXANEb5jbPGQ83uHC+Q=
```

Reply:
That's really odd, I must have done something wrong when I tested it locally.. Anyway, I could reproduce it and have now updated the hash.

Thanks!